PanModal/Controller/PanModalPresentationController.swift

```diff
@@ -234,12 +246,57 @@ open class PanModalPresentationController: UIPresentationController {
             }
         })
     }
-
+    
+    // MARK: - Actions
+    
+    /**
+     Update long form height and short form height when the keyboard is showing/scrolling and scroll to correct position above the keyboard if keyboard is showing
+     */
+    @objc func keyboardWillShow(notification: Notification) {
+        
+        //if the presented view is already at max height and is scrollable then scroll to above the keyboard when keyboard is shown if needed
+        
+        if let keyboardSize = (notification.userInfo?  [UIResponder.keyboardFrameEndUserInfoKey] as? NSValue)?.cgRectValue {
+            
+            if let scrollView = presentable?.panScrollable, pannedToMax {
+                scrollView.contentInset = UIEdgeInsets(top: 0.0, left: 0.0, bottom: keyboardSize.height, right: 0.0);
```

Comment:
This makes assumptions that there is no content inset for the scrollview in the first place.